README.md

```diff
@@ -31,3 +31,6 @@ To get a local copy up and running, follow these steps
      ```
 4. Run `npm start` to start Play Reading Party.
 5. Navigate to http://localhost:3000 in your browser
+
+### Deployment
```

Comment:
Should the README be updated to link to the deployed site as well?

Reply:
Add to github's about section instead